<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainframeClass</class>
 <widget class="QDialog" name="mainframeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainframe</string>
  </property>
  <widget class="QGroupBox" name="groupBoxSerialSetting">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>144</width>
     <height>295</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial Port Setting:</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Baudrate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxBaud">
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSerialPort"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Stop bit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxStopBit">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Data bit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDataBit">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Check bit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCheckBit">
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ODD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MARK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPACE</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainframe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>mainframeClass</receiver>
   <slot>Start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Start()</slot>
 </slots>
</ui>
